pages/ExpenditurePage.js

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+import {AiTwotoneHome} from "react-icons/ai";
+import Link from "next/link";
+
+export default function Expenditure() {
+  return (
+    <Header1>
+      <Link className="Homebutton" href="/">
+        <AiTwotoneHome />
+      </Link>
+      Detailed Expenditure
+    </Header1>
+  );
+}
+const Header1 = styled.h1`
+  border: 1px solid;
+  margin: 0;
+  text-align: center;
+
+  .Homebutton {
+    position: absolute;
+    margin: 0;
+    left: 0px;
```

Comment:
You could write it like this:
```suggestion
    left: 0;
```